packages/react-styled-ui/src/Drawer/Drawer.js

```diff
@@ -90,6 +90,7 @@ const Drawer = ({
     <DrawerProvider value={drawerState}>
       <Portal container={mountRef.current}>
         <FocusLock
```

Comment:
After offline discussion, please remove `<FocusLock />` component from `Drawer` component. And add an example for `Focus Lock Drawer`.

Reply:
@tinaClin 
Just hold a moment to figure out a better solution rather than making a big change to remove the `<FocusLock>` component.

Reply:
https://next.material-ui.com/components/trap-focus/

I suggest adding a new prop `ensureFocus` (or `trapFocus`) to disable the focus lock and set the default value to `false`.

Reply:
We can also apply the same change to the Modal component.